graphitron-codegen-parent/graphitron-java-codegen/src/test/resources/queries/fetch/interfaces/multitableInterface/paginated/expected/QueryDBQueries.java

```diff
@@ -44,32 +44,29 @@ public static List<Pair<String, SomeInterface>> someInterfaceForQuery(DSLContext
         var mappedCustomer = customerForSomeInterface();
 
         return ctx.select(
-                        unionKeysQuery.field("$type"),
-                        mappedAddress.field(1),
-                        mappedCustomer.field(1))
+                        DSL.row(
+                                unionKeysQuery.field("$type", String.class),
+                                mappedAddress.field("$data"),
+                                mappedCustomer.field("$data")
```

Comment:
Tror jeg kanskje må gå tilbake til å gette med indeks her igjen. Prøvde å ta bort siden det gikk bra hos oss, men ser jeg fikk en aliasfeil i SIS. (oracle/jooq problem?)

```suggestion
                                mappedAddress.field(1),
                                mappedCustomer.field(1)
```